MetaU Capstone Project/View Controllers/MusicSessionViewController.m

```diff
@@ -41,27 +39,31 @@ @implementation MusicSessionViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    [[NSNotificationCenter defaultCenter] addObserver:self
+            selector:@selector(receiveNotification:)
+            name:@"playerStateChangeNotification"
+            object:nil];
+    
     self.accessToken = [[SpotifyManager shared] accessToken];
     self.sessionNameLabel.text = self.musicSession.sessionName;
     self.sessionIDLabel.text = self.musicSession.sessionCode;
-    self.isPlaying = NO;
+    self.isPlaying = YES;
     
     [self querySetup];
+    [self updateView];
     
-    [MusicSession addUserToSession:self.musicSession.sessionCode withCompletion:^(BOOL succeeded, NSError * error) {
-        if (error != nil) {
-            NSLog(@"Error: %@", error.localizedDescription);
-        } else {
-            NSLog(@"User added to session successfully");
-        }
-    }];
-    
-    [self getDataFrom:@"https://api.spotify.com/v1/me/player/recently-played"];
-//    NS
+    [MusicSession addUserToSession:self.musicSession.sessionCode withCompletion:nil];
```

Comment:
I think it would be a good idea to keep the completion block to handle any errors.
Maybe you want to show an alert if it fails?
Anyway, just a suggestion